Specs/Scene/ExpressionSpec.js

```diff
@@ -13,6 +13,33 @@ defineSuite([
     MockStyleEngine.prototype.makeDirty = function() {
     };
 
+    function MockFeature() {
+        this._properties = {};
+    }
+
+    MockFeature.prototype.addProperty = function(name, value) {
+        this._properties[name] = value;
+    };
+
+    MockFeature.prototype.getProperty = function(name) {
+        return this._properties[name];
+    };
+
+    it('evaluates variable', function() {
+        var feature = new MockFeature();
+        feature.addProperty('height', 10);
```

Comment:
See how it works out with Cesium's `Color` object.  We can add `addColorProperty` if we need to, but we might just have to do some `typeof` checks to handle colors cleanly from the API perspective.